protocols/gossipsub/src/behaviour.rs

```diff
@@ -930,6 +930,15 @@ where
         }
     }
 
+    /// Returns a scoring parameters for a topic if existent.
+    pub fn get_topic_params<H: Hasher>(&self, topic: &Topic<H>) -> Option<TopicScoreParams> {
```

Comment:
should be enough to return a reference. If the caller needs a value they can clone it